h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -58,8 +66,11 @@ public PersistStatsEntry() {
 
   public static boolean isHdfsPath(String path) {
     String s = path.toLowerCase();
-    if (s.startsWith("hdfs:") || s.startsWith("s3:") || s.startsWith("s3n:")
-                          || s.startsWith("s3a:") || s.startsWith("maprfs:")) {
+    if (s.startsWith("hdfs:")
+        || s.startsWith("s3:")
+        || s.startsWith("s3n:")
+        || s.startsWith("s3a:")
+        || s.startsWith("maprfs:")) {
```

Comment:
Good point, originally i wanted to remove this method and corresponding paths from PersistManager API. However, will keep it for in for simplicity of this commit.